test/db/cmd/cmd_pf2

```diff
@@ -496,7 +496,6 @@ EXPECT=<<EOF
      shnum : 0x00000030 = 0
   shstrndx : 0x00000032 = 0
 EOF
-EXPECT_ERR=
```

Comment:
FYI, this is because the `r_core_bin_export_info_rad` is generating an eprintf. This was always generated, however without this PR rabin2 is directly called and the stderr of rabin2 is not caught, so the eprintf was never actually printed. With this PR we directly use the right api function, however I could not find an easy way to ignore stderr (and probably we should not ignore it, but we should fix somehow the code in `r_core_bin_export_info_rad`.

That however goes beyond my current knowledge of the `pf` code and this PR anyway fixes the linked issue and it improves compared to master because it does not rely on calling another binary.

Reply:
ok